src/components/recordedit/form-row.tsx

```diff
@@ -125,6 +128,24 @@ const FormRow = ({
     };
   }, []);
 
+  if (columnModelIndex === 0) {
```

Comment:
I think it's better if you move this check to be inside the `useEffect` line 139. React itself says that we should avoid doing something like this.